providers/stores/mongo.js

```diff
@@ -49,18 +50,50 @@ class MongoStore {
    * @param {Coordinates} coordinates - The coordinates of the object to get
    * @returns The loaded object
    */
-  get(coordinates) {
-    return this.collection.findOne({ _id: this._getId(coordinates) }, { projection: { _id: 0 } })
+  async get(coordinates) {
+    const cursor = await this.collection.find(
+      { _id: new RegExp(`^${this._getId(coordinates)}`) },
+      { projection: { _id: 0, _mongo: 0 }, sort: { _id: 1 } }
```

Comment:
will this sorting work for docs that have more than 10 pages?
```
../../1
../../10
../../11
../../2
../../3
```
perhaps we should just get the all the pages, then `sortBy(.. mongo.currentPage`) and `flatten`

Reply:
I found that I can sort by `_mongo.page` on the server and it respects number order
sorting on the database would be better I think. 

I also updated the queries to be by partitionKey instead of regexing the IDs